evalml/tests/component_tests/test_imputer.py

```diff
@@ -350,18 +385,22 @@ def test_imputer_woodwork_custom_overrides_returned_by_components(X_df, has_nan,
         X_df.iloc[len(X_df) - 1, 0] = np.nan
     override_types = [Integer, Double, Categorical, NaturalLanguage, Boolean]
     for logical_type in override_types:
+        # Converting a column with NaNs to Boolean will impute NaNs.
```

Comment:
In the old version of woodwork, the conversion of nullable dtype to nullable bool would fail. Here it doesn't so I'm skipping those test cases. This is not a loss of test coverage.